.env

```
@@ -0,0 +1,13 @@
+  type= "service_account"
+  project_id= "express-sheets-demo"
+  private_key_id= "sample-private-key-id"
+  private_key= "sample-private-key"
+  client_email= "sample-client-email"
+  client_id= "sample-client-id"
+  auth_uri= "sample-auth-uri"
+  token_uri= "sample-token-uri"
+  auth_provider_x509_cert_url= "sample-auth-provider-cert"
+  client_x509_cert_url = "sample-client-cert"
+  sheetId= "sample-sheetid"
```

inconsistent casing
```suggestion
  sheet_id= "sample-sheetid"
```
Or, alternatively, maybe make everything camelCase

---

All the other keys like **project_id** or **auth_provider_x509_cert_url** are downloaded from the google developers console.
I don't think we should change them; so to be consistent I will make it as **sheet_id**

@mderazon What do you think?

---

Okay